src/workspaces/cookiefactoryv2/README.md

```diff
@@ -162,7 +193,7 @@ For any issue not addressed here, please open an issue or contact AWS Support.
    sh cloud9\_resize.sh 20
    df -h  
    ```
-### No space during `cdk deploy`: `OSError: [Errno 28] No space left on device` 
+### No space during `cdk deploy: OSError: [Errno 28] No space left on device` 
```

Comment:
fyi that the 2 backtick sections was intentional here, the former was the command and the latter was the error message, but I don't feel strongly if you feel merging into 1 block is more readable

Reply:
ah, sorry. i must've changed accidentally.